src/components/Form/Form.js

```diff
@@ -906,24 +910,48 @@ class Form extends Component {
             case "instructor": {
                 let instructorList = this.props.instructors;
 
-                instructorList = Object.values(instructorList).map(({user_id, name, email}) => ({
-                    value: user_id,
-                    label: `${name} - ${email}`,
-                }));
-                instructorList = this.removeDuplicates(Object.values(this.state[label]), instructorList);
-                return (<div style={{width: "inherit"}}>
-                    <Grid container className="student-align">
-                        <SearchSelect
-                            disabled={disabled}
-                            value={this.state[label][fieldTitle] ? this.state[label][fieldTitle] : ""}
-                            onChange={(value) => {
-                                this.onSelectChange(value, label, field);
-                            }}
-                            placeholder={"Choose an Instructor"}
-                            options={instructorList}
-                            className="search-options" />
-                    </Grid>
-                </div>);
+                const mapInstructor = (id, name, email) => ({
+                    "value": id,
+                    "label": `${name} - ${email}`,
+                })
+
+                instructorList = Object.values(instructorList)
+                    .map(({user_id, name, email}) => mapInstructor(user_id, name, email));
+
+                const loadInstructors = async (inputValue) => {
```

Comment:
Let's move the async function out of Form.js since Form.js is really unweildy already. Move it to FormUtils.js

Reply:
I can't remove `loadInstructors` from Form.js unless Form is a functional component - it relies on `this.props.authToken`, which I need Redux to get.